src/main/integrations/preload-injection.ts

```diff
@@ -49,7 +50,7 @@ export class PreloadInjection implements Integration {
       //
     }
 
-    if (path && typeof path === 'string' && existsSync(path)) {
+    if (path && typeof path === 'string' && isAbsolute(path) && existsSync(path)) {
```

Comment:
Can we log out an error if the path is not absolute? Or atleast attempt to log out the path in the debug message below?

Reply:
I would be wary of logging any warnings here since all this change actually does it stop Electron from logging warnings about preload non-absolute paths. 

If injecting preload fails, we fallback to custom protocol + `fetch` so the SDK should continue to function as normal.

Reply:
Ah ok that makes sense, thanks for the details!